Libplanet/Action/ActionEvaluator.cs

```diff
@@ -483,9 +483,10 @@ ActionContext CreateActionContext(IAccountStateDelta prevStates, int randomSeed)
                     .ForContext("Tag", "Metric")
                     .ForContext("Subtag", "TxEvaluationDuration")
                     .Debug(
-                        "{ActionCount} actions in transaction {TxId} by {Signer} " +
+                        "{ActionCount} {ActionType} actions in transaction {TxId} by {Signer} " +
```

Comment:
```suggestion
                        "{ActionCount} actions {ActionType} in transaction {TxId} by {Signer} " +
```

This would be more natural.